api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -95,66 +85,29 @@ public DbInstitution setOrganizationTypeOtherText(String organizationTypeOtherTe
     return this;
   }
 
-  @OneToMany(mappedBy = "institution", cascade = CascadeType.ALL)
```

Comment:
Remove this error prone logic.  It's now distributed among mappers, InstitutionServiceImpl, and the DbInstitutionEmailDomain/DbInstitutionEmailAddress classes.